RELEASE.rst

```diff
@@ -0,0 +1,13 @@
+RELEASE_TYPE: minor
+
+This release introduces two new features:
+
+* pytest users can specify a seed to use for ``@given`` based tests by passing
+  the ``--hypothesis-seed`` command line argument.
+* When a test fails, either with a health check failure or a falsifying example,
+  Hypothesis will print out a seed that lead to that failure if the test is not
```

Comment:
"lead" -> "led" or "leads"; or just reread the whole sentence and decide on a tense.